pkg/server/server.go

```diff
@@ -171,10 +169,9 @@ func (s *Server) StreamedListObjects(req *openfgapb.StreamedListObjectsRequest,
 }
 
 func (s *Server) Read(ctx context.Context, req *openfgapb.ReadRequest) (*openfgapb.ReadResponse, error) {
-	store := req.GetStoreId()
+	storeID := req.GetStoreId()
```

Comment:
Half are `store` and the other half `storeID`. Changes to be consistent, as least in this file.